schemas/WebpackOptions.json

```diff
@@ -1484,7 +1484,16 @@
         },
         "runtimeChunk": {
           "description": "Create an additional chunk which contains only the webpack runtime and chunk hash maps",
-          "type": "boolean"
+          "oneOf": [
+            {
+              "type": "boolean"
+            },
+            {
+              "enum": [
+                "single"
```

Comment:
Please also add the counterpart

Reply:
Do you mean I should add another string option called `multiple`?

Reply:
Yes.